packages/discord.js/src/structures/InteractionCollector.js

```diff
@@ -6,14 +6,14 @@ const Events = require('../util/Events');
 
 /**
  * @typedef {CollectorOptions} InteractionCollectorOptions
- * @property {TextBasedChannelResolvable} [channel] The channel to listen to interactions from
  * @property {ComponentType} [componentType] The type of component to listen for
  * @property {GuildResolvable} [guild] The guild to listen to interactions from
  * @property {InteractionType} [interactionType] The type of interaction to listen for
  * @property {number} [max] The maximum total amount of interactions to collect
  * @property {number} [maxComponents] The maximum number of components to collect
  * @property {number} [maxUsers] The maximum number of users to interact
- * @property {Message|APIMessage} [message] The message to listen to interactions from
+ * @property {Snowflake} [channelId] The channel id of the message interaction response
+ * @property {Snowflake} [messageInteractionId] The message with a given interaction id to listen to interactions from
```

Comment:
Is this a message id, or an interaction id? The description only confused me further

>The message

So its a message id?

>with a given interaction id 

So I'm giving an interaction id now?

And either way, messageId is still a property, but I don't see where this can be separately provided as an option anymore since you removed the `message` option

Reply:
Oops I accidentally removed `message` and `channel`